apps/ex_shop/web/models/line_item.ex

```diff
@@ -44,6 +49,33 @@ defmodule ExShop.LineItem do
     cast(model, Map.merge(params, %{total: cost}), ~w(total), ~w())
   end
 
+  def move_stock(%ExShop.LineItem{fullfilled: true} = line_item) do
+    remove_stock_from_variant(line_item)
```

Comment:
Can we sync the 2 methods like remove_stock_from_variant and add_stock_to_variant

or 

take_stock_from_variant / move_stock_back_to_variant ??